### DISCUSSION TITLE
options for default `na.strings`?

### DISCUSSION THREAD

**Discussion by olivroy** [JanMarvin/openxlsx2#964] [General]
I almost always want `na.strings = ""` when writing data.

I know there is a waiver function, but I don't really understand how it works. 

Thanks

**Comment by JanMarvin** [comment]
Might be a good idea. The waiver function is simply a blocker that indicates internally that the equivalent of `NA` is used. Initially it was checking for `missing(na.strings)`, but this caused issues with Jordan’s huge wbWorkbook splitting approach in #471 iirc. Our waiver is just a custom `rlang::missing_arg`.

It should be straightforward to check if some global option is set.

**Comment by olivroy** [comment]
Implemented as of #968
